### DISCUSSION TITLE
Rejection for no reason

### DISCUSSION THREAD

**Discussion by EloiD-R** [community/community#134411] [GitHub Education] [answered]
### Hi everyone,

Hi, i just applied bc the application i did in 2022 expired, so i entered all infos and proofs, the only thing is that i'm not on campus bc it's vacations here in France. First try was rejected and told me to complete my billing infos, i did it, re-applied, it still doesn't work, i tried one more time but no more success, but i actually still match all requirements so :/

**Comment by PakmanGames** [comment]
Hey, make sure the document you upload is valid proof. It should include your full legal name (make sure it matches your name in your billing settings) and it should have a date. Try applying with a letter of proof of enrollment from your school or a valid student ID with a date on it.

**Reply by EloiD-R** [reply]
i did it, still doesn't work, i tried again today but it's the same, i just don't know what to do, i've even asked the assistant on the support but it wasn't helpful, the ony thing is still that i'm not on campus but i was already accepted in 2022 outside of campus, i just can't understand

**Reply by ravilovrm** [reply]
Im from Russia, Saint-Petersburg, and i have the same problem!

**Reply by EloiD-R** [reply]
Well, i just asked on the github embassy in the hackclub slack, hoping for someone to respond something useful

**Reply by EloiD-R** [reply, marked as answer]
somebody from github responded and helped me now i have it !!,  @TUTBOROMA  she said that i needed to provide the translated version of the official document just next to it, hope it can help!
